app/client/src/components/Footer/index.tsx

```diff
@@ -157,7 +157,10 @@ export default function Footer(props: { showStat: boolean }) {
 			? analyticsData.chaosOperatorCount.toString()
 			: "0";
 	const githubStars = formatCount(githubData.star_count);
-	const expRuns = formatCount(analyticsData.totalExpRuns);
+	const expRuns =
+		analyticsData.totalExpRuns != undefined
+			? analyticsData.totalExpRuns.toString()
+			: "0";
```

Comment:
Put this in a single line

Reply:
This is due to the prettier configurations, on saving it will again be formatted as such, it cant be changed without changing the prettier configuration across entire frontend. PTAL at the present code in this file. @rajdas98